src/meltano/core/venv_service.py

```diff
@@ -228,8 +303,17 @@ async def create(self) -> Process:
         """
         logger.debug(f"Creating virtual environment for '{self.namespace}/{self.name}'")
         try:
-            return await exec_async(sys.executable, "-m", "virtualenv", str(self.venv))
```

Comment:
Why not use the `--python` option here?



[-p], [--python] | the python executable virtualenv is installed into | interpreter based on what to create environment (path/identifier) - by default use the interpreter where the tool is installed - first found wins
-- | -- | --

[-p]: https://virtualenv.pypa.io/en/latest/cli_interface.html#p
[--python]: https://virtualenv.pypa.io/en/latest/cli_interface.html#python

https://virtualenv.pypa.io/en/latest/cli_interface.html#section-discovery

Reply:
I wasn't aware of this option. This lets us avoid having to fall back to using the built-in `venv` instead of `virtualenv`. Thanks!

Reply:
Addressed in 4a69f8bf5a2d7153d51b16ac4f5d05e6de8feed7